objects/cash_register.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+# This is the CashRegister class
+class CashRegister
+  attr_accessor :purchase, :total, :pay
```

Comment:
You don't need an attr_accessor for purchase and pay. Remember, the attr_accessor is like writing a method to access instance variables like:
```
def total
  @total
end
```

Since you don't have instance variables called purchase and pay, you don't need to ever access them. Your purchase and pay methods do different things